makefile

```diff
@@ -13,6 +13,12 @@ build:
 	@echo "Building $(GOFILES) to ./bin"
 	@GOBIN=$(GOBIN) go build -mod=mod -o bin/$(GONAME) $(GOFILES) 
 
+buildaws:
+	@GOBIN=$(GOBIN) GOARCH=amd64 GOOS=linux go build -mod=mod -o bin/application $(GOFILES) 
+	@cd web && yarn build
+	@-rm bin/ebUpload.zip
+	@zip -r bin/ebUpload.zip bin/application web/build
+
```

Comment:
creates the artifacts which I'm currently uploading manually to aws elastic beanstalk